test/jdk/com/sun/jdi/JdwpNetProps.java

```diff
@@ -67,19 +67,26 @@ public static void main(String[] args) throws Exception {
             new ListenTest("localhost", ipv4Address)
                     .preferIPv4Stack(false)
                     .run(TestResult.Success);
+            new ListenTest("localhost", ipv4Address)
+                    .preferIPv4Stack(true)
+                    .preferIPv6Addresses("system")
+                    .run(TestResult.Success);
             if (ipv6Address != null) {
                 // - only IPv4, so connection prom IPv6 should fail
```

Comment:
Is there an existing typo here, I assume "prom" should be "from"?

Reply:
I agree: that is pre-existing, and it is a typo for 'from'. I am happy to fix it while I'm here if you like.